x-pack/plugins/infra/public/components/asset_details/tabs/overview/alerts.tsx

```diff
@@ -105,36 +86,3 @@ export const AlertsSummaryContent = ({
     </>
   );
 };
-
-interface MemoAlertSummaryWidgetProps {
-  alertsQuery: AlertsEsQuery;
-  dateRange: TimeRange;
-  onLoaded: (alertsCount?: AlertsCount) => void;
-}
-
-const MemoAlertSummaryWidget = React.memo(
-  ({ alertsQuery, dateRange, onLoaded }: MemoAlertSummaryWidgetProps) => {
-    const { services } = useKibanaContextForPlugin();
-
-    const summaryTimeRange = useSummaryTimeRange(dateRange);
-
-    const { charts, triggersActionsUi } = services;
-    const { getAlertSummaryWidget: AlertSummaryWidget } = triggersActionsUi;
-
-    const chartProps = {
-      baseTheme: charts.theme.useChartsBaseTheme(),
-    };
-
-    return (
-      <AlertSummaryWidget
-        chartProps={chartProps}
-        featureIds={infraAlertFeatureIds}
-        filter={alertsQuery}
-        timeRange={summaryTimeRange}
-        onLoaded={onLoaded}
-        fullSize
-        hideChart
```

Comment:
`hideChart` was introduced to the `AlertSummaryWidget` for this use case here so it can be removed from the widget if it's not needed anymore (it can be a follow-up PR after this one is merged if it's too much effort).

Reply:
Do you mean `AlertSummaryWidget` from `triggersActionsUi` ? I think it can be useful to have the option to hide the chart, it makes the component more configurable for the consumer. 

Do you think we need to remove it?

Reply:
I added it in the `triggersActionsUi`'s `AlertSummaryWidget`  because we wanted to have the widget in `fullSize` without the chart (and without the compact design style changes) and it was maybe too specific and probably not useful anymore. We can also decide to leave it in case someone else needs similar customization. Up to you 🙂